schema/composite.gql

```diff
@@ -68,7 +68,7 @@ type CompositeResourceSpec {
   """
   The resources of which this composite resource is composed.
   """
-  resources: KubernetesResourceConnection @goField(forceResolver: true)
+  resources: KubernetesResourceConnection! @goField(forceResolver: true)
```

Comment:
How does this work if an XR actually doesn't have any composed resources? It it just saying it has to have a connection, but the connection may have zero resources?

Reply:
that's correct, the nodes in connection are optional (this is true for all connections) and total will be 0